### PR TITLE
[Feature] Add UNet benchmark with multiple losses supervision.

### PR DESCRIPTION
**Pull request by MengzhangLI**
## Motivation
To improve numerical results of UNet benchmark, using multiple losses (1.0 * cross entropy loss + 3.0 * dice loss) is introduced in this pr.

To testify representation ability of UNet on natural image, benchmark of UNet on Cityscapes dataset is also added in this pr.

## Note
(1) For `DRIVE`, `STARE`, `CHASE_DB1` and `HRF`, using multiple losses (dice vs cross entropy is 3:1).
(2) Add cityscapes benchmark.

- [x] Upload models & logs
- [x] Keep all models using `CrossEntropyLoss` only and using multiple losses with `CrossEntropyLoss` and `DiceLoss`.
- [x] Add `mDice` results of existing using `CrossEntropyLoss` only models.

### CODE REVIEW

**Comment by Junjun2016** [comment]
Should rename the title to descript this PR in more detail.

**Comment by Junjun2016** [comment]
Should not override the previous results.
Key previous results and add new results.

**Review by MeowZheng — APPROVED** [review]

**Comment by Junjun2016** [comment]
> Should rename the title to descript this PR in more detail.

Should resolve this comment. :)

**Review by Junjun2016 — COMMENTED** [review]
The config with multiple losses can inherit from the config with only CE loss.

**Review by Junjun2016 — APPROVED** [review]
LGTM
